live-server.js

```diff
@@ -116,20 +116,17 @@ for (var i = process.argv.length - 1; i >= 2; --i) {
 	}
 }
 
-if (process.argv[2]) {
-	process.chdir(process.argv[2]);
-}
-
 // Patch paths
-var cwd = process.cwd();
+var dir = opts.root = process.argv[2] || null;
```

Comment:
This would overwrite `opts.root` given from config file. Also potentially gives null to path.join, which seems to result in TypeError.